lib/travis/api/v3/services/user/update.rb

```diff
@@ -0,0 +1,17 @@
+module Travis::API::V3
+  class Services::User::Update < Service
+    result_type :user
+    params :utm_params
+
+    def run!
+      raise LoginRequired unless access_control.logged_in? && access_control.user
+
+      Models::UserUtmParam.new(
```

Comment:
Nice 👍 I just have one question though, maybe I am wrong but didn't we decide to have separate columns rather than saving the data as string? https://github.com/travis-ci/travis-migrations/pull/217#issuecomment-598269268

Reply:
Hey @shairyar, thank for your comment! Later in that thread we decided that one column should work fine, but it should be of JSON format instead of string (https://github.com/travis-ci/travis-migrations/pull/217#issuecomment-598281761). Does it make sense?

Reply:
Yes, thanks for clarifying